book/content/binderhub/binderhub.md

```diff
@@ -0,0 +1,159 @@
+
+Binder, as a user interface, is useful for reproducibility because the code needs to be version controlled and the computational environment needs to be documented in order to benefit from the functionality of Binder.
+Each change to the code repository also forces a new build of the Binder instance.
+This acts as a proxy for continuous integration of the computational environment as the Binder instance will break if the configuration file is not updated.
+
+Learn more about Continuous Integration in [this chapter](/continuous_integration/continuous_integration).
+
+## How does a BinderHub work?
+
+BinderHub relies on different tools and resources in order to create and launch the Binder instances.
+
+A high-level explanation of the BinderHub architecture can be found [here](https://binderhub.readthedocs.io/en/latest/overview.html).
+
+| ![cloud_neutral_binderhub](/figures/cloud_neutral_binderhub.png) |
+|:---:|
+| A representation of the BinderHub architecture. |
+
+### Compute Resources
+
+BinderHub is cloud-neutral which means it can be deployed on any cloud platform.
+Therefore, the minimum requirement is a subscription to a cloud platform of your choosing.
+
+In fact, BinderHub is not dependent on cloud-hosting at all and can be deployed onto an on-premise computing system.
+
+### Kubernetes
+
+[Kubernetes](https://kubernetes.io/) is a system for automating deployment, scaling (making more or fewer copies), and management of containers across a compute cluster (it doesn't have to be cloud-based).
+BinderHub uses Kubernetes to manage the resources requested by the users of the Binder service, and to support the tools of that build the instances.
+
+### Helm
+
+[Helm](https://helm.sh/) is a package manager for Kubernetes.
+Packages come in the form of Charts which are a set of instructions to deploy, upgrade and manage applications running on a Kubernetes cluster.
+They can make installing and managing Kubernetes applications much easier and Specific Charts for projects can be published online.
+For example, the Helm Chart for BinderHub are available [here](https://jupyterhub.github.io/helm-chart/#development-releases-binderhub).
+
+### JupyterHub
+
+[JupyterHub](https://jupyter.org/hub) is a multi-user server for Jupyter Notebooks and containers alike.
+The JupyterHub's main role is to connect the user's browser to the BinderHub instance running on the Kubernetes cluster.
+However, the JupyterHub can be further customised to provide greater control over the operation of the BinderHub.
+
+### repo2docker
+
+[repo2docker](https://repo2docker.readthedocs.io/en/latest/?badge=latest) is a tool that automatically builds a Docker image from a code repository given a configuration file.
+This Docker container will contain all of the code, data and resources that are listed in the repository.
+All the software required to run the code will also be preinstalled from the configuration file.
```

Comment:
Just a note here - I often describe BinderHub as a thin layer that sits on top of repo2docker and JupyterHub.

Reply:
Addressed in [`ccaed45`](https://github.com/alan-turing-institute/the-turing-way/pull/557/commits/ccaed45f74beb26ff0623dfa37f86965ada7947f)